### ISSUE TITLE
Local (Non-Archive) Builds Should Have Differentiated Product Names and Icons

### ISSUE THREAD

**Issue by stuartbreckenridge** [Ranchero-Software/NetNewsWire#3854]
This will assist in testing (e.g., at the moment there's no way to differentiate a NetNewsWire production install from a NetNewsWire debug build in System Settings).

At first glance, we will need to do this in `xcconfig`, in addition to considering any changes to the DeveloperSettings instructions and script.

**Comment by brentsimmons** [comment]
I put this on the 6.3 milestone. Let’s consider it at that time. It needs some discussion — the gains are obvious, but what do we lose by doing this?

**Comment by vincode-io** [comment]
I don't think we lose anything by doing it. I think it is a pretty common industry practice.

**Comment by stuartbreckenridge** [comment]
I lose the rag regularly because we _haven’t_ done this.
